mlflow/tracking/client.py

```diff
@@ -915,7 +922,24 @@ def print_run_info(r):
             tags: {'t': 't'}
             status: FINISHED
         """
-        self._tracking_client.log_batch(run_id, metrics, params, tags)
```

Comment:
@nzw0301 Apologies for the confusion. `mlflow.tracking.MlflowClient` is a thin wrapper around `mlflow.tracking._tracking_service.client.TrackingServiceClient`. Can we move this implementation to `mlflow.tracking._tracking_service.client.TrackingServiceClient` instead?

Reply:
Sure!